tool/site/client/src/components/common/rawApiResponse.tsx

```diff
@@ -73,10 +73,10 @@ const RawApiResponse: React.FC<RawApiResponseProps> = ({
           </SummaryList.Row>
         </SummaryList>
         {response.headers && (
-          <XmlMessageExpander name="Response Headers" message={formatAsString(response.headers)} mimeType="text/plain"/>
+          <XmlMessageExpander name="Response Headers" message={formatAsString(response.headers)}/>
```

Comment:
this used to be `text/plain`, now its `text/xml`, is this a problem?

Reply:
Just done some testing, this is continuing to work fine